Model/Feed.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Taxjar_SalesTax
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Taxjar
+ * @package    Taxjar_SalesTax
+ * @copyright  Copyright (c) 2017 TaxJar. TaxJar is a trademark of TPS Unlimited, Inc. (http://www.taxjar.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+
+namespace Taxjar\SalesTax\Model;
+
+
+
+class Feed extends \Magento\AdminNotification\Model\Feed
+{
+    /**
+     * Retrieve feed url
+     *
+     * @return string
+     */
+    public function getFeedUrl()
+    {
+        $httpPath = $this->_backendConfig->isSetFlag(self::XML_USE_HTTPS_PATH) ? 'https://' : 'http://';
+        if ($this->_feedUrl === null) {
+            $this->_feedUrl = $httpPath . \Taxjar\SalesTax\Model\Configuration::TAXJAR_FEED_URL;
```

Comment:
Let's do `use Taxjar\SalesTax\Model\Configuration as TaxjarConfig;` at the top of this file to shorten this to `TaxjarConfig::TAXJAR_FEED_URL`